PilotDataPlatform/arranger
# PILOT-5954: Update CI pipelines to use semantic versioning and new ACR

## Summary

- Delete old CI pipelines
- Update new CI pipelines to use new `pilot_develop` branch
- Update new CI pipelines to use new ACR and semantic versioning
- Trivy vulnerabilities identified are out of scope of this PR and will be addressed in future PR. Documented vulnerabilities here: https://indocconsortium.atlassian.net/wiki/spaces/PILOT/pages/4021846025/Arranger+.trivyignore+CVE+justification

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-5954

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Verify arranger admin-server, admin-ui, and server-filter and build and publish to new ACR with semantic version of `1.1.0`
